pywinauto/application.py

```diff
@@ -839,6 +841,15 @@ def __connect(self, **kwargs):
     def connect(self, **kwargs):
         """Connects to an already running process"""
         connected = False
+
+        if 'timeout' in kwargs:
+            try:
+                self.timeout = float(kwargs['timeout'])
+                del kwargs['timeout']
+                WaitUntilPasses(self.timeout, Timings.app_connect_retry, partial(self.Connect, **kwargs), (ValueError))

```

Comment:
You may suggest how to do it? I have not found how to avoid "connect () takes one positional argument but two were given"